### DISCUSSION TITLE
Annotation suggestions

### DISCUSSION THREAD

**Discussion by pulsar123** [sumatrapdfreader/sumatrapdf#3257] [Ideas]
Hi,

First of all - thank you so much for this amazing program! I just updated it (to 3.4.6) and discovered the new great feature - annotations. Would it be possible to implement some of the following improvements?

 - Keyboard shortcuts to jump to the next (and previous) annotations.
 - Ability to make the Annotation editor window skinnier. Right now I can make it pretty skinny, but once I click on any of the annotations in the list, the window becomes wider ( I think it adjusts to the width of the button "Save changes to existing PDF"). Even better - perhaps this can be made a (narrow) sidebar of the main window? This is for ease of navigation between annotations, when screen estate is at premium. This can be a separate feature - not Annotations editor, but Annotations browser (or navigator).

**Comment by GitHubRulesOK** [comment]
- up down key should cause main window to follow annotations (or click entry with mouse) , however there is no current way to re-order after insertion.
- it floats to be able to set it to either side (or outside), and generally keeps whatever shape (within reason) you drag to

![image](https://user-images.githubusercontent.com/12754913/217105033-123bf694-dd90-4533-85c6-88ca14f8f037.png)

**Comment by pulsar123** [comment]
Thanks for the fast feedback! What I meant was keyboard shortcuts which would let me navigate between annotations without opening the annotation editor. In my current setup I don't have space for the Annotations editor, so being able to say press Crtl + -> to go to the next annotation (while in the main window) would be really great.

**Comment by GitHubRulesOK** [comment]
Long Time ago in an app built for SumatraPDF legacy annotations there, was such a (very good) feature but same as here the developer had limited time opportunity and payment for expanding in every new development twist and turn

Currently I see the easy current year option as keep window small on one side or the other and call topmost with either use right click above any annotation or say e button, then dont close simply focus on new location, so it drops behind.
```
Shortcuts [
  [
  cmd = CmdEditAnnotations
  key = e 
  ]
]
```

**Comment by ndoylend** [comment]
The shortcut suggested by @GitHubRulesOK works well. It would be nice if it were possible to dock the annotations window on the right hand side of the main window, or on the left, swapping between showing bookmarks and annotations. It would also be good if the software didn't prompt to save annotations on exit when all the user has done is used the annotation editor to navigate annotations.

**Reply by GitHubRulesOK** [reply]
@ndoylend  
> if the software didn't prompt to save annotations on exit  

that is a different issue where if an existing annotation is disturbed it triggers as a change so intentional or not
it is important to prompt, 
However the other request is when reviewing annotation without any intent to change
then the editor should be inactive by some setting.
In some cases simply clicking on annotation can destroy a 3rd party custom annotation.
